src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -361,7 +362,7 @@
     <Compile Include="Microsoft\Data\SqlClient\SessionHandle.Unix.cs" />
   </ItemGroup>
   <!-- Assets needed on Windows but should be avoided on UAP to avoid sni.dll -->
-  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(IsPartialFacadeAssembly)' != 'true' and '$(IsUAPAssembly)' != 'true'">
+  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' and '$(IsUAPAssembly)' != 'true' ">
```

Comment:
Nit: There is white space padding around the condition which does not match the convention of most other conditions.